website.js

```diff
@@ -261,6 +261,18 @@ app.get('/history', function(req, res) {
     res.render("history.html")
 });
 
+app.get('/poll', function(req, res) {
+    connection.query('select * from poll', function(err, result) {
+        res.render('poll.html')
+    });
+});
+
+app.get('/poll/:id', function(req, res) {
+    connection.query('select * from poll', function(err, result) {
```

Comment:
A lot of duplicated code (here and on line `#265`). Shouldn't we put this in separate functions?

```javascript
// The render function that takes the `response` as a parameter and returns
// a new function. This function can then be passed to the `findAllPolls` function
function renderPoll(res) {
  return function(err, result) {
    res.render('poll.html');
  }
}

// The function that does a query to the database and takes a callback
// function as a parameter so that any other function can decide what to
// do with the result.
function findAllPolls(callback) {
  connection.query('select * from poll', callback);
}

// Then in your app:
app.get('/poll', function(req, res) {
  findAllPolls(renderPoll(res));
});

app.get('/poll/:id', function(req, res) {
  findAllPolls(renderPoll(res));
});
```

You can even do it like this:

```javascript
function findAllPolls(callback) {
  connection.query('select * from poll', callback);
}

function renderPolls(req, res) {
  findAllPolls(function(err, result) {
    res.render('poll.html');
  });
}

// In your app
app.get('/poll', renderPolls);
app.get('/poll/:id', renderPolls);

```